vcap/vcap/device_mapping.py

```diff
@@ -112,7 +112,7 @@ def filter_func(devices):
                 "OPENVINO_ALLOWABLE_DEVICES", None)
 
             allowed_devices = []
-            if allowed_device_type in (None, ""):
+            if allowed_device_type.lower() in (None, ""):
```

Comment:
why is `.lower()` necessary to compare to `None` and an empty string?

Reply:
In fact, if `allowed_device_type` is `None`, this will throw an exception

Reply:
Oh nevermind I reviewed the wrong commit somehow

Reply:
Yeah I don't know what I was thinking lol